qiskit/visualization/matplotlib.py

```diff
@@ -1059,7 +1059,10 @@ def _draw_ops(self, verbose=False):
             self._xmax = self._fold + 1 + self._x_offset - 0.9
             self._ymax = (n_fold + 1) * (self._n_lines + 1) - 1
         else:
-            self._xmax = max_anc + 1 + self._x_offset - 0.9
+            if self._ops == []:
+                self._xmax = max_anc + 1 + self._x_offset - 0.4
+            else:
+                self._xmax = max_anc + 1 + self._x_offset - 0.9
```

Comment:
```
            x_incr = 0.4 if not self._ops else 0.9
            self._xmax = max_anc + 1 + self._x_offset + x_incr
```
Looks good. Typically we use `if not self._ops` in this situation and what I have here is a bit more concise.

Reply:
Thanks! I've made the changes. :)